Sources/SignHereLibrary/Commands/CreateProvisioningProfileCommand.swift

```diff
@@ -258,23 +259,12 @@ internal struct CreateProvisioningProfileCommand: ParsableCommand {
             throw Error.unableToBase64DecodeProfile(name: profileResponse.data.attributes.name)
         }
 
-    //    let fileManager = FileManager.default
-        let filePath = "\(outputPath)/\(profileResponse.data.attributes.uuid).mobileprovision"
-        print(filePath)
-        // do{
-        //     if !fileManager.fileExists(atPath: outputPath) {
-        //         try fileManager.createDirectory(atPath: outputPath, withIntermediateDirectories: true, attributes: nil)
-        // }
+        let filePath = "\(outputPath)/\(bundleIdentifier)-\(profileType).mobileprovision"
```

Comment:
Using a uuid for the _file_name is what we want here. We don't want to get in the game of managing overwrites/deletes on every update. If you look in `~/Library/MobileDevice/Provisioning Profiles`, Xcode also names developer profiles with globally unique names. Both Xcode (which we will no longer "use") and rules_apple are good at finding the _newest_ profile applicable to an app and using that:

https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-apple.md#local_provisioning_profile

Reply:
But the original code would write to `--outputPath`, so this doesn't seem like it's even necessary to decide in this method? It should ultimately be written to that path.

Reply:
> Using a uuid for the _file_name is what we want here.

Roger that. I made this change prior to confirming it doesn't matter and then didn't revert.